### PR TITLE
This pull adds Responsive Header, Footer and the Editor View

### PR DESCRIPTION
**Pull request by ogbiyoyosky**
To make this fully functional, a new implementation of search from the header and check category from the header has to be implemented. According to the UI, Notifications also should also be implemented
image

reference #34

### CODE REVIEW

**Comment by the-codingninja** [comment]
Sorry but the latest merge has created conflicts can you please resolve it @ogbiyoyosky 

Thanks

**Review by the-codingninja — COMMENTED** [review]
The PR looks good code quality wise

Need to test the functionality

Will check once the conflicts are resolved

**Comment by ogbiyoyosky** [comment]
The conflicts have been resolved.

File: src/components/Header/index.js [hunk 1]
```diff
@@ -1,48 +1,160 @@
...
-          {user.username && <Link to="/new"><Button type="primary">New Contribution</Button></Link>}
-          {!user.username && <a href={getOathURL()}><Button>Login</Button></a>}
-          {user.username && <Link to="/"><Button onClick={onLogoutClick}>Logout</Button></Link>}
-          {user.username && <Tooltip title={user.username}><a href={`https://steemit.com/@${user.username}`}><div className="avatar" style={{backgroundImage: `url(https://steemitimages.com/u/${user.username}/avatar)`}} /></a></Tooltip>}
-        </div>
-      </div>
-    </Header>
-  );
+
+=======
```
Comment (the-codingninja) [inline src/components/Header/index.js]: This got left while resolving conflicts

Comment (ogbiyoyosky) [reply src/components/Header/index.js]: Should be all good now

**Review by the-codingninja — APPROVED** [review]

**Pull request merged**